src/level/level.js

```diff
@@ -118,6 +130,37 @@ Level.prototype.validateLevel = function() {
 		validateObject(this.world, template.world, true);
 		validateObject(this.gameplay, template.gameplay, true);
 
+		// Validate texture properties
+		if (this.textures) {
```

Comment:
This property (along with the materials one) should at least exist as an empty object, so this should throw an error if it doesn't.

Reply:
Yes, *if* it exists, that is. It's not a vital level property, right?

Reply:
So something like `if (this.textures && typeof this.textures === "object") {`